src/components/ProjectSettingsRedirect.tsx

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from "@apollo/client";
+import { Redirect } from "react-router-dom";
+import {
+  getProjectSettingsRoute,
+  ProjectSettingsTabRoutes,
+} from "constants/routes";
+import { GetViewableProjectRefsQuery } from "gql/generated/types";
+import { GET_VIEWABLE_PROJECTS } from "gql/queries";
+
+export const ProjectSettingsRedirect: React.FC = () => {
+  const { data } = useQuery<GetViewableProjectRefsQuery>(GET_VIEWABLE_PROJECTS);
```

Comment:
nit. Even though it does not have any variables, Can you also include the types for the variables. Just so we can guarantee type safety for this query.